projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import JetpackLogo from '../jetpack-logo';
+import JetpackFooter from '../jetpack-footer';
+
+/**
+ * This is the base structure for any admin page. It comes with Header and Footer.
+ *
+ * All content must be passed as children wrapped in as many <JetpackAdminSection> elements as needed.
+ *
+ * @param {object} props - Component properties.
+ * @returns {React.Component} JetpackAdminPage component.
+ */
+const JetpackAdminPage = props => {
+	const { children, moduleName, a8cLogoHref } = props;
+
+	return (
+		<>
+			<div className="jp-admin-page-section">
+				<div className="jp-wrap">
+					<div class="jp-row">
+						<div class="lg-col-span-12 md-col-span-8 sm-col-span-4">
+							<JetpackLogo />
+						</div>
+					</div>
+				</div>
+			</div>
```

Comment:
That was the idea at first, but later I saw little benefit in doing that.

Especially because we are about to refactor the whole CSS structure, and this would change anyway. I'm only concerned with the public API right now (the JetpackAdminPage element and its props).

But if I was to do it, I'd actually create a Header and a Footer internal components, and the component would be basically:

```
<Header />
{ children }
<Footer />
```

Reply:
Done in https://github.com/Automattic/jetpack/pull/21826/commits/2a1dfa7d64730974c7991458fff8575d263d8cf2